stregsystem/static/stregsystem/gdpr.js

```diff
@@ -0,0 +1,16 @@
+/*
+Hides the username after 5 seconds of showing it.�
+Simply replaces the letters with 'x'.
+*/
+
+var username_element = document.getElementById("username");
+if (username_element !== null) {
+
+  setTimeout(function () {
+
+    var username_element = document.getElementById("username");
+    if (username_element !== null) {
+      username_element.innerText = "x".repeat(username_element.innerText.length);
```

Comment:
```suggestion
      username_element.innerText = "xxxxxxxx";
```

If we're anonymising, then we might as well do it properly. I suggest statically printing `xxxxxxxx` instead.

_Also, I have an idea for a meme on this. I'll PR that later_